src/main/drivers/dma.h

```diff
@@ -22,6 +22,8 @@
 
 #include "resource.h"
 
+//#ifdef USE_DMA // Should be gone?
```

Comment:
Can we just not add these?

Reply:
Dam*!!! I did a build test and forgot to completely remove it. Can you re-approve it after the brewing is done?